quacc/calculators/qchem.py

```diff
@@ -0,0 +1,177 @@
+"""
+A Q-Chem calculator built on Pymatgen and Custodian functionality
+"""
+from __future__ import annotations
+
+import inspect
+import os
+import struct
+
+from ase import Atoms, units
+from ase.calculators.calculator import FileIOCalculator
+from monty.io import zopen
+from pymatgen.io.ase import AseAtomsAdaptor
+from pymatgen.io.qchem.outputs import QCOutput
+from pymatgen.io.qchem.sets import ForceSet
+
+from quacc.custodian import qchem as custodian_qchem
+
+
+class QChem(FileIOCalculator):
+    """
+
+    Parameters
+    ----------
+    input_atoms
+        The Atoms object to be used for the calculation.
+    cores
+        Number of cores to use for the Q-Chem calculation.
+    charge
+        The total charge of the molecular system.
+        Effectively defaults to zero.
+    spin_multiplicity
+        The spin multiplicity of the molecular system.
+        Effectively defaults to the lowest spin state given the molecular structure and charge.
+    qchem_input_params
+        Dictionary of Q-Chem input parameters to be passed to pymatgen's ForceSet.
```

Comment:
When possible, would you mind referring to the specific module in the docstring where possible to help if someone is just inspecting the class with hover-over in VS Code? i.e. "passed to `pymatgen.io.qchem.sets.ForceSet`"